### DISCUSSION TITLE
zsh: killed

### DISCUSSION THREAD

**Discussion by Mr-x2020** [projectdiscovery/naabu#366] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/65666323/174778424-c5aa40b7-c57b-4e31-a371-af8fde99a083.png)

any solution for that issue?

**Comment by ehsandeep** [comment, marked as answer]
@Mr-x2020 I would suggest updating naabu to the latest version at first, but in this case, the issue is about running out of memory, whats' the target input count?

**Reply by Mr-x2020** [reply]
Actually, my file size is 47mb 
![image](https://user-images.githubusercontent.com/65666323/174885647-3ade0a26-fe26-4b6a-8c83-3e5f9f3ef67c.png)
can you make it deal with such size?
